gradle.properties

```diff
@@ -30,5 +30,5 @@ POM_DEVELOPER_ORGANIZATION=Amazon Web Services
 POM_DEVELOPER_ORGANIZATION_URL=http://aws.amazon.com
 
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
This i'm not too sure about.. once I upgraded to roboeletric 4.7.1, i got this error:

```

```

Related issue: https://github.com/robolectric/robolectric/issues/6521

What's the implications of disabling Jetifer? what should be the correct change?

Reply:
From stackoverflow:
```
Jetifier helps in migrating an android project to AndroidX.

Jetifier helps in making 3rd party library compatible with your AndroidX project.

If you're not using any 3rd party library in your android project, then you don't need to use Jetifier 
because Jetifier helps only in "making 3rd party library" compatible with your project in 
case you're migrating to AndroidX. So to make your project use Jetifier, you write below 
code in gradle.properties.

android.enableJetifier=true
android.useAndroidX=true

If you're not migrating to AndroidX and using the previous way of using Android libraries
(e.g: com.android.support), then these 3rd party libraries need not be made compatible 
as these 3rd party libraries are already compatible with the previous way of using Android libraries. 
In that case, you don't need to use Jetifier, So now, you don't need to write lines given above or 
you can simply write as below to change the values in future to move to androidx:-

```

So I guess the critical question is if we have any 3rd party libraries that are pointing to the old location.  Given that all of the tests pass when this is turned off, that would indicate to me we don't.  I will try to look back to where it was turned on and see if there was a specific reason for it.

Reply:
Jetifier was enabled in this release: https://github.com/aws-amplify/amplify-android/commit/e4eb992564e47bc34d4f76940d2880ec343f8fcd 

That was over 3 years ago.  A lot of time for our dependencies to be updated.   The fact that all of the tests pass is a very good sign that this is ok.

Reply:
I found a limited fix that allows us to block just the offending library.  This should be much safer than turning off jetifier altogether.  Pushed to the branch.

Reply:
thanks for looking into this, it sounds like the impact of disabling it would only impact the amplify developers writing tests and not to the developers using the library, so given the existing tests are passing, it seems like a low risk change to make. The disallow list also makes sense and is even better then